python_modules/dagster-test/dagster_test/fixtures/docker_compose.py

```diff
@@ -89,11 +89,9 @@ def docker_compose_down(docker_compose_yml, context, service, env_file):
         compose_command += ["--env-file", env_file]
 
     if service:
-        compose_command += ["--file", str(docker_compose_yml), "rm", "--volumes"]
-
-        compose_command.append(service)
-
+        compose_command += ["--file", str(docker_compose_yml), "down", "--volumes", service]
```

Comment:
Is this functionally different? Or just cleaning up weird syntax?

Reply:
`rm` only removes stopped containers so was a no-op 

`down` will actually stop the specific service and remove the container

Reply:
Ah - I missed the shift from `rm` -> `down`